frontend-vue-js/src/plugins/router.js

```diff
@@ -67,6 +68,11 @@ const routes = [
                 component: PharmacistLanding,
                 name: 'pharmacist',
                 path: ''
+            },
+            {
+                component: PharmacistProfile,
+                name: 'pharmacistProfile',
+                path: '/pharmacistProfile'
```

Comment:
This path will already have pharmacist in it so it's currently '/pharmacist/pharmacistProfile', wouldn't just '/pharmacist/profile' make more sense?

Reply:
Of sure, nice, i will fix that

Reply:
Fixed